### ISSUE TITLE
Live reload

### ISSUE THREAD

**Issue by TatriX** [seed-rs/seed#172]
It looks like currently `seed` has no live reload feature. Do you have this planned and can I somehow help with it?

**Comment by MartinKavik** [comment]
It would be nice to have it in [default quickstart](https://github.com/David-OConnor/seed-quickstart) (I don't know if some crates exist for this or we have to write something from scratch). 
Or you can use [webpack quickstart](https://github.com/MartinKavik/seed-quickstart-webpack) (I will try to update it and make it better in next weeks / month but it should be usable now).

**Comment by TatriX** [comment]
Having rust-only solution is a more pleasing alternative ;)

**Comment by MartinKavik** [comment]
Yeah, my hands were burning while I was writing Javascript for Rust project, but assets minifications, Tailwind, live reload, CSS transformations, etc. are still missing in Rust and I don't have time to write them all now :D

**Comment by MartinKavik** [comment]
I'll create something like Rust Webpack for Seed once a style library for Seed is complete. And I want to investigate [elm-spa](https://github.com/ryannhg/elm-spa/) for inspiration.

I'm closing this issue in favor of https://github.com/seed-rs/seed-quickstart/issues/11 (where you can add your ideas).
